migrations/0013_add_process_column_to_ports.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE ports ADD COLUMN taken text;
+UPDATE ports SET taken = 't' WHERE port = (SELECT port FROM ports WHERE app_id != NULL);
```

Comment:
This should also set app_id to null. This migration should probably remove that column, since it'll be unused after this change.